configuration/coding_plans.py

```diff
@@ -27,6 +27,24 @@ def clean_district_if_no_mogadishu_sub_district(text):
         return Codes.NOT_CODED
 
 
+def clean_post(post):
```

Comment:
```suggestion
def clean_facebook_post_type(post):
```

Reply:
Thanks, this is much more descriptive. Updated